frame/pov-limit/Cargo.toml

```
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-pov-limit"
```

```suggestion
name = "pallet-weight-limit"
```
Maybe to be more generic?

---

We could even use an actual name, like *glutton* or something.

---

Sounds funny :P

---

I actually quite like "Glutton Parachains". It has the double meaning of one who over-consumes (the parachain), but also one who relishes in enduring something (i.e. "glutton for punishment"), and it is seeing how much validators can endure from parachains.

I also didn't know this before but it's another name for wolverines (https://en.wikipedia.org/wiki/Wolverine), so it can have a nice logo :).